test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/CheckClaimantDetailsControllerSpec.scala

```diff
@@ -186,12 +197,13 @@ class CheckClaimantDetailsControllerSpec
             val consignee  = acc14.displayResponseDetail.consigneeDetails.getOrElse(fail())
             //Registered Details with CDS
             paragraphs.get(0).text() shouldBe consignee.legalName
-            paragraphs.get(1).text() shouldBe fillingOutClaim.signedInUserDetails.verifiedEmail.value
-            paragraphs.get(2).text() shouldBe consignee.establishmentAddress.addressLine1
-            paragraphs.get(3).text() shouldBe consignee.establishmentAddress.addressLine2.getOrElse(fail)
-            paragraphs.get(4).text() shouldBe consignee.establishmentAddress.addressLine3.getOrElse(fail)
-            paragraphs.get(5).text() shouldBe consignee.establishmentAddress.postalCode.getOrElse(fail)
-            paragraphs.size()        shouldBe 6
+            paragraphs.get(1).text() shouldBe consignee.contactDetails.flatMap(_.telephone).getOrElse(fail)
+            paragraphs.get(2).text() shouldBe fillingOutClaim.signedInUserDetails.verifiedEmail.value
+            paragraphs.get(3).text() shouldBe consignee.establishmentAddress.addressLine1
+            paragraphs.get(4).text() shouldBe consignee.establishmentAddress.addressLine2.getOrElse(fail)
+            paragraphs.get(5).text() shouldBe consignee.establishmentAddress.addressLine3.getOrElse(fail)
+            paragraphs.get(6).text() shouldBe consignee.establishmentAddress.postalCode.getOrElse(fail)
+            paragraphs.size()        shouldBe 7
```

Comment:
Also never liked multiple assertions. Always thought that there are better ways of doing it - https://www.scalatest.org/scaladoc/1.5/org/scalatest/matchers/BeMatcher.html
But it just legacy code what has nothing to do with you

Reply:
This looks longer than these individual matcher